js/ftx.js

```diff
@@ -615,26 +612,26 @@ module.exports = class ftx extends Exchange {
                 'strike': undefined,
                 'optionType': undefined,
                 'precision': {
-                    'amount': sizeIncrement,
-                    'price': priceIncrement,
+                    'price': this.safeNumber (market, 'priceIncrement'),
+                    'amount': this.safeNumber (market, 'sizeIncrement'),
                 },
                 'limits': {
+                    'leverage': {
+                        'min': this.parseNumber ('1'),
+                        'max': this.parseNumber ('20'),
+                    },
                     'amount': {
-                        'min': sizeIncrement,
+                        'min': undefined,
                         'max': undefined,
                     },
                     'price': {
-                        'min': priceIncrement,
+                        'min': undefined,
```

Comment:
Should this and amount.min be changed to undefined?

Reply:
I think, the minimal amount is 1 contract for contracts and undefined for spot markets, and the minimal price is undefined for all of them, but that has to be confirmed by testing different contracts.